### ISSUE TITLE
Simpler way of implementing

### ISSUE THREAD

**Issue by FouseyTube** [OraOpenSource/apex-diff#21]
This looks promising but can we have a simpler way of making it working.
Like i have installed node,sqlcl, dowloaded zip files(extracted it as well)  from here and made changes in config.sql file accordingly to my credentials but whenever i run the command node \...\app.js dev 113it only opens app.js file.
Second where to get application file to convert into json format i have downloaded it from my local env. but do we need to put it somewhere.

**Comment by Sam-Hall** [comment]
I'm looking for something like apex-diff that can simply process an application export file without the need to connect to the database at all.

**Comment by martindsouza** [comment]
@Sam-Hall I don't think that currently exists. Next best thing is to try APEXcl: https://apexcl.dev/
